colosseum/declaration.py

```diff
@@ -276,22 +317,22 @@ def __init__(self, **style):
 
     # 15. Fonts ##########################################################
     # 15.3 Font family
-    # font_family
+    font_family = validated_property('font_family', choices=FONT_FAMILY_CHOICES, initial=INITIAL)  # TODO: initial?
```

Comment:
font_family isn't a "normal" property, because it can be multivalued. The single string representation is fine for output/debug purposes, but end consumers of a style really need the list of pre-parsed font names, rather than a string that needs to be repeatedly parsed.

Reply:
Will fix!

Reply:
@freakboy3742 would it be better to make this a tuple? or how should we handle mutability here, if the user pops a value from the list? Should we return a copy of the list?

Reply:
@freakboy3742 what do you mean by 

>this isn't a tuple - position isn't significant, only ordering.

Reply:
This is one of those theoretical aspects of API design that isn't always followed well because it's not obvious that it exists. The underlying question - why is there two "ordered" datatypes in Python?

Consider the coordinates for a point, and a path made up of a collection of points.

A point is a *tuple* of three values - x, y and z. It must *always* be three values - you can't add a new value; and there's significance to each position - the second value *is* the coordinate on the y axis.

A path is a *list*. The list can be of arbitrary length, and you can add new values to the end of the list. However, there's no particular significance to the each position, other than the ordering - there's nothing especially significant about position 2, other than the fact that it comes after position 1, and before position 3 (if it exists).

There's one other notable difference: every value in a list *should* be the same data type (or, at least, have some common duck-base). There's no such requirement of a tuple. In the case of point, they all happen to be coordinates - but consider something like an RGBA color specification. RGB are integers/hex values; A is a percentage. 

A tuple isn't just an immutable list. Immutability is a useful property when all the positions are significant, but it's not the most interesting distinguishing property between a List and a Tuple. 

Put another way - If Python were to remove index-based addressing from lists, it should be *inconvenient*, but not problematic, because lists should be iterated over. Tuples, on the other hand, aren't generally interesting to iterate over, because you need to handle each position differently. In practice, this isn't *completely* true, because indexing lists can be useful when iterating, and good functional programming means you can use positional based function dispatch, but hopefully my underlying point is clear.

Reply:
Perfectly clear now :-), thanks for the thorough explanation